src/frontend/planner_test/tests/testdata/array.yaml

```diff
@@ -72,13 +72,14 @@
     └─BatchValues { rows: [[]] }
 - sql: |
     select array_cat(array[233], array[array[array[66]]]);
-  binder_error: "Bind error: unable to find least restrictive type between integer[] and integer[][][]"
+  binder_error: 'Bind error: unable to find least restrictive type between integer[]
```

Comment:
Styling differences generated by `./risedev do-apply-planner-test`. Not sure why they were introduced by #5345